Sources/variadics-generator/VariadicsGenerator.swift

```diff
@@ -80,258 +80,54 @@ func outputForEach<C: Collection>(
 }
 
 struct VariadicsGenerator: ParsableCommand {
-  @Flag(help: "Generate `Zip's")
```

Comment:
Removed the unnecessary variadic bits and moved all the `PathZip`s into `PathBuilder`, which also cleans up how docs are rendered.